libshaderc/src/shaderc.cc

```diff
@@ -279,7 +279,7 @@ shaderc_util::Compiler::TargetEnv GetCompilerTargetEnv(shaderc_target_env env) {
     case shaderc_target_env_opengl_compat:
       return shaderc_util::Compiler::TargetEnv::OpenGLCompat;
     case shaderc_target_env_webgpu:
-      return shaderc_util::Compiler::TargetEnv::WebGPU;
+      assert(false);
```

Comment:
This ends up being an "unannotated fallthrough".

Reply:
Just sent up a fix for this